zenoh/src/net/runtime/mod.rs

```diff
@@ -110,6 +110,7 @@ impl Runtime {
             .unwrap_or(false);
 
         let gossip = config.scouting().gossip().enabled().unwrap_or(true);
+        let gossip_multihop = config.scouting().gossip().multihop().unwrap_or(false);
```

Comment:
A generic comment: `zenoh-config` crate has a [default.rs](https://github.com/eclipse-zenoh/zenoh/blob/master/commons/zenoh-config/src/defaults.rs) file containing all default values. 

Wouldn't be better to use the `Default` implementation for the config instead of manually setting the value to `false`?

Reply:
We've had this `Default` debate multiple times, using options helps distinguish whether the value is set by user or left at default behaviour. This may be useful when merging configuration, or when other configuration fields may change the default value for a given field.

Reply:
I think the point of @Mallets is not to remove the `Option` but to centralise the hard coded defaults into the default.rs file instead of spreader around in the code. A agree with @Mallets but let's deal with this in another PR.